client/my-sites/checkout/composite-checkout/components/checkout-help-link.tsx

```diff
@@ -0,0 +1,204 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import Gridicon from 'calypso/components/gridicon';
+import styled from '@emotion/styled';
+import { keyframes } from '@emotion/core';
+import { get } from 'lodash';
+import { useEvents, useLineItemsOfType } from '@automattic/composite-checkout';
+import type { LineItem } from '@automattic/composite-checkout';
+
+/**
+ * Internal dependencies
+ */
+import { HappychatButton as HappychatButtonUnstyled } from 'calypso/components/happychat/button';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import getSupportLevel from 'calypso/state/selectors/get-support-level';
+import isPresalesChatAvailable from 'calypso/state/happychat/selectors/is-presales-chat-available';
+import isHappychatAvailable from 'calypso/state/happychat/selectors/is-happychat-available';
+import isSupportVariationDetermined from 'calypso/state/selectors/is-support-variation-determined';
+import QuerySupportTypes from 'calypso/blocks/inline-help/inline-help-query-support-types';
+import { showInlineHelpPopover } from 'calypso/state/inline-help/actions';
+import getSupportVariation, {
+	SUPPORT_HAPPYCHAT,
+	SUPPORT_FORUM,
+	SUPPORT_DIRECTLY,
+} from 'calypso/state/selectors/get-inline-help-support-variation';
+import {
+	isWpComBusinessPlan,
+	isWpComEcommercePlan,
+	isWpComPersonalPlan,
+	isWpComPremiumPlan,
+} from 'calypso/lib/plans';
+
+type StyledProps = {
+	theme?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
+};
+
+const HappychatButton = styled( HappychatButtonUnstyled )`
+	margin: 0;
+	padding: 0;
+
+	svg {
+		margin-right: 6px;
+		width: 20px;
+
+		.rtl & {
+			margin-right: 0;
+			margin-left: 6px;
+		}
+	}
+`;
+
+export function PaymentChatButton( { plan }: { plan: string | undefined } ): JSX.Element {
+	const reduxDispatch = useDispatch();
+	const translate = useTranslate();
+	const supportLevel = useSelector( getSupportLevel );
+
+	const chatButtonClicked = () => {
+		reduxDispatch(
+			recordTracksEvent( 'calypso_presales_chat_click', {
+				plan,
+				support_level: supportLevel,
+			} )
+		);
+	};
+
+	return (
+		<HappychatButton onClick={ chatButtonClicked }>
+			<Gridicon icon="chat" />
+			{ translate( 'Need help? Chat with us.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **29** times:
`translate( 'Need help? Chat with us' )` *ES Score: 10*